src/js/languages/default.json

```diff
@@ -86,6 +86,10 @@
     "openMenu": "Open Menu",
     "closeMenu": "Close Menu"
   },
+  "pagination": {
+    "stepOptions": "Items per page",
```

Comment:
@MikeKingdom do you think this should be called `itemsPerPage` to be more close to the default string?

Reply:
I'd go with something like `stepLabel`. It helps more to know it's a field label

Reply:
that makes sense I like `stepLabel`

Reply:
updated 👍